### ISSUE TITLE
how do you feel about upload of curent openmeeg's master into debian/neurodebian?

### ISSUE THREAD

**Issue by yarikoptic** [openmeeg/openmeeg#103]
besides that failing test for python (and didn't check API changes etc) #101 and installing additional module I thought to skip #102, seems to build/test fine (at least on amd64)

thought to version it as `2.3.0~20160502` i.e. version before 2.3.0 as of today... any alternative?

**Comment by agramfort** [comment]
>
> besides that failing test for python (and didn't check API changes etc)
> #101 <https://github.com/openmeeg/openmeeg/issues/101> and installing
> additional module I thought to skip #102
> <https://github.com/openmeeg/openmeeg/issues/102>, seems to build/test
> fine (at least on amd64)
>
thought to version it as 2.3.0~20160502 i.e. version before 2.3.0 as of
> today... any alternative?
>
that's a good way forward.

thx !

**Comment by papadop** [comment]
Yes. I hope we will have a new "real" release during the summer....

**Comment by agramfort** [comment]
even before :)

**Comment by yarikoptic** [comment]
FWIW -- I did upload to debian unstable (with disabled python test iirc) and amazingly it worked out almost everywhere!!! 
https://buildd.debian.org/status/package.php?p=openmeeg&suite=unstable
need to deal only with s390x where tests failed for some "special" reason

**Comment by papadop** [comment]
Yes. Reply to another post, I may have seen the same log. To repeat what I said in issue #75:
Things seem to mostly work now. Except for S390 and PPC64, And to some extend Hurd (but in this case all test pass, so this seems more a packaging problem as .so files seems to not be found). For S290 and PPC64, it is difficult to analyze without a machine at hand....

**Comment by yarikoptic** [comment]
nevermind those -- they are not officially supported ports yet (grayed out) so would be not as important. s390x is the one to take care about since we had it built there successfully before:
https://buildd.debian.org/status/logs.php?pkg=openmeeg&arch=s390x . I can troubleshoot on such a box, just need to know what to look for ;)

**Comment by papadop** [comment]
Well strangely I do not find the tests on the amd64 build. For s390x, it seems there is a problem with reading a matrix. Test 22 is the first to fail. The very first check is to see whether the file Head1.sgmm exists (/«PKGBUILDDIR»/obj-s390x-linux-gnu/OpenMEEG/build/tests/Head1.sgmm). If it exists can you give access to it (to see if we can read it on another machine). You can also try a om_matrix_info 
sipadan-> tools/om_matrix_info -i /user/papadop/home/compiles/OM-test/OpenMEEG/build/tests/Head1.sgmm
tools/om_matrix_info version 2.3.dev (Unversioned directory) compiled at May 11 2016 17:47:27

Loading : /user/papadop/home/compiles/OM-test/OpenMEEG/build/tests/Head1.sgmm
Dimensions : 162 x 42
Min Value : -0.000596554 (12,16)
Max Value : 0.000596554 (43,21)
First Values
1.91057e-18 -3.21206e-15 -2.76018e-15 -8.80021e-18 -9.00921e-08 
-3.21904e-15 1.36851e-17 -8.76884e-18 -2.77804e-15 9.00921e-08 
-2.77812e-15 -8.95717e-18 1.38735e-17 -3.20942e-15 2.13985e-07 
-8.92606e-18 -2.76159e-15 -3.21066e-15 -9.76881e-18 -2.13985e-07 
1.01038e-06 -1.01038e-06 -1.72517e-07 1.72517e-07 -3.34104e-17 

If it does not exist or if it cannot be read, we have to look at the test that generates this file.

By the way, I read that s390x is a big endian machine (like sparc that failed in a report earlier), so it might be this and then it will be nice to have access to the machine. My main suspect would be matio, but I suppose that the matio test suite passed, so....
